src/settings.py

```diff
@@ -325,6 +326,7 @@
 PING_IF_NUMS_ACCS = {}
 
 is_role = lambda plyr, rol: rol in ROLES and plyr in ROLES[rol]
+start_votes_required = lambda num_players: min((num_players + 1) / 2.2, 4)
```

Comment:
Prefer actual settings rather than lambdas-as-settings, even if it means you need to break it into more than one. Settings should really be scalars. Also, settings should be ALL_UPPERCASE. If this isn't meant to be a setting, don't put it in settings.py.